lib/services/serviceBus/README.md

```diff
@@ -14,9 +14,66 @@ npm install azure-sb
 
 ## How to Use
 
-### Authentication
+```node
+var azure = require('azure');
```

Comment:
it is unclear this code has nothing to do with the Authentication, should we remove it?